.dockerignore

```diff
@@ -14,6 +14,12 @@ private/buf/cmd/buf/command/alpha/protoc/internal/protoc-gen-insertion-point-rec
 private/buf/cmd/buf/command/alpha/protoc/internal/protoc-gen-insertion-point-writer/protoc-gen-insertion-point-writer
 private/buf/cmd/buf/command/alpha/protoc/test.txt
 private/buf/cmd/buf/command/generate/internal/protoc-gen-top-level-type-names-yaml/protoc-gen-top-level-type-names-yaml
+private/buf/cmd/buf/internal/buf-plugin-panic/cmd/buf-plugin-panic/buf-plugin-panic
```

Comment:
- Some of the plugins are used by `bufcheck`, which is a dependency of `private/buf/cmd/buf`
- I don't see why `duplicate-plugins` should be grouped in their own subdirectory while the othders shouldnt
- The `Specs` in each test plugin aren't actually used outside of the `main` (I think you patterned this off of the examples, which I just changed)

I would:

- Move these to `private/bufpkg/bufcheck/internal/cmd/buf-plugin-.*`
- Flatten the duplicate plugins to the same level as the others
- Make them each just be a `main` package, ie instead of doing `internal/buf-plugin-panic/cmd/buf-plugin-panic`, just do `internal/cmd/buf-plugin-panic`

Reply:
Plugins have been moved and collapsed accordingly, and did a merge with `bufplugin` to pick-up latest updates.